README.md

```diff
@@ -45,7 +45,7 @@ And here are links of other repositories:
 
 ## Contributing
 
-- Welcome to contribute to HugeGraph, please see [`How to Contribute`](CONTRIBUTING.md) & [Guidelines](https://hugegraph.apache.org/docs/contribution-guidelines/) for more information.  
+- Welcome to contribute to HugeGraph, please see [How to Contribute](https://github.com/apache/incubator-hugegraph/blob/master/CONTRIBUTING.md) for more information.  
```

Comment:
seems we should leave the website as the latest version instead of keeping hugegraph-server's doc